<launch>
        <!-->without visualize node <-->
        <!-->with remap ir/rgb node <-->
	<include file="$(find livox_ros_driver)/launch/livox_lidar_msg.launch">
	</include>
	<include file="$(find hikvision_ros)/launch/hik.launch">
	</include>
	<include file="$(find usb_cam)/launch/usb_flir-test.launch">
	</include>
	<node pkg="collect_data" type="ir_rgb_remap" name="ir_rgb_remap"  respawn="false" output="screen"></node>
        <!--include file="$(find color_pc)/launch/fusion.launch">
	</include-->
</launch>
